product/utilities/color/index.md

```diff
@@ -16,240 +16,300 @@ layout: app
 <p class="m-bottom-4">The DreamHost palette is primarily derived from blue and white. The secondary palette consists of red, green, orange, and yellow. These colors are used across all web apps at DreamHost. Also included are dedicated background and grayscale classes which are derived from blue.</p>
 
 <h3 class="m-bottom-3">Primary Palette</h3>
-<div class="u-clearfix m-bottom-7">
-<div class="g-1_2 g-1_4__l bg-c-b700 p-top-12 m-bottom-2 m-bottom-0__l">
-	<ul class="t-center p-2 u-clearfix">
-			<li class="t-center t-c-w100 g-1_4"><span class="t-1">A</span><br />&#128515;</li>
-			<li class="t-center t-c-w100 g-1_4"><span class="t-0">A</span><br />&#128515;</li>
-			<li class="t-center t-c-b700 g-1_4"><span class="t-1">A</span><br />&#128557;</li>
-			<li class="t-center t-c-b700 g-1_4"><span class="t-0">A</span><br />&#128557;</li>
-		</ul>
-		<p class="p-3 bg-c-g100 m-0"><strong>Midnight</strong><br />Sass $c-b700<br />Hex #071C26<br />rgb(0,115,236)</p>
-	</div><!--Midnight-->
-	<div class="g-1_2 g-omega g-1_4__l bg-c-b300 p-top-12 m-bottom-2 m-bottom-0__l">
-		<ul class="t-center p-2 u-clearfix">
-			<li class="t-center t-c-w100 g-1_4"><span class="t-1">A</span><br />&#128515;</li>
-			<li class="t-center t-c-w100 g-1_4"><span class="t-0">A</span><br />&#128515;</li>
-			<li class="t-center g-1_4 t-c-b700"><span class="t-1">A</span><br />&#128515;</li>
-			<li class="t-center g-1_4 t-c-b700"><span class="t-0">A</span><br />&#128515;</li>
-		</ul>
-		<p class="p-3 bg-c-g100 m-0"><strong>Azure</strong><br />Sass $c-b300<br />Hex #0073EC<br />rgb(0,175,239)</p>
-	</div><!--Azure-->
-	<div class="g-1_2 g-1_4__l bg-c-p300 p-top-12 m-bottom-2 m-bottom-0__l">
-		<ul class="t-center p-2 u-clearfix">
-			<li class="t-center t-c-w100 g-1_4"><span class="t-1">A</span><br />&#128515;</li>
-			<li class="t-center t-c-w100 g-1_4"><span class="t-0">A</span><br />&#128515;</li>
-			<li class="t-center g-1_4 t-c-b700"><span class="t-1">A</span><br />&#128515;</li>
-			<li class="t-center g-1_4 t-c-b700"><span class="t-0">A</span><br />&#128515;</li>
-		</ul>
-		<p class="p-3 bg-c-g100 m-0"><strong>Prince</strong><br />Sass $c-p300<br />Hex #A644E5<br />rgb(166,68,229)</p>
-	</div><!--Prince-->
-	<div class="g-1_2 g-omega g-1_4__l bg-c-w100 p-top-12 m-bottom-2 m-bottom-0__l">
-		<ul class="t-center p-2 u-clearfix">
-			<li class="t-center t-c-w100 g-1_4"><span class="t-1">A</span><br />&#128557;</li>
-			<li class="t-center t-c-w100 g-1_4"><span class="t-0">A</span><br />&#128557;</li>
-			<li class="t-center g-1_4 t-c-b700"><span class="t-1">A</span><br />&#128515;</li>
-			<li class="t-center g-1_4 t-c-b700"><span class="t-0">A</span><br />&#128515;</li>
-		</ul>
-		<p class="p-3 bg-c-g100 m-0"><strong>White</strong><br />Sass $c-w100<br />Hex #FFFFFF<br />rgb(255,255,255)</p>
-	</div><!--White-->
+<div class="container p-0 m-bottom-2">
```

Comment:
Not gonna mark all the containers haha, but please check them on this page and change to `container-fluid` where needed.

Reply:
Ok sounds good. I'll go through the suggested changes and start updating. Thank you so much!

Reply:
I went through the requested changes and used container-fluid where it made sense for alignment. Tested flexing in gulp environment to make sure nothing looked wonky in transition. Let me know if there's anything else. Thanks again!